packages/networks/src/networks/sepolia.ts

```diff
@@ -47,6 +47,12 @@ export const sepolia: NetworkConfig = {
         id: HookType.PROMO_CODE_CAPPED,
         name: 'Discount code with caps',
       },
+      {
+        // should be replaced with unlock's deployed contract address
+        address: '0xD92546cE22f13304C41e92d18D4f000Ac295aDb7',
```

Comment:
The address should be replaced with the one obtained from deploying the hook in Unlock's own environment (if necessary).

Reply:
I would actually be fine using yours! I see this is verified! Great!
did you deploy this with a script?

Reply:
okay great. yes i did. here's the repo containing the deployment script, plus hardhat configuration:

https://github.com/0xTxbi/gitcoin-passport-hook

Reply:
Hum I am confused! There is a deployment script in your other PR https://github.com/unlock-protocol/unlock/pull/13426/files#diff-59353e5f06d8c1661abc0040d558e7e2402019ea1e5cea06bd60d692a2184320
Did you use that one?

Please make sure you test with that script you created because this is the one we should use to deploy all the networks :)

Reply:
yes I used that one. also tested with the script created, and it passed